SDL/Event.cu

```diff
@@ -560,10 +562,14 @@ public:
         int const & nLowerModules) const
     {
 
-        int const globalThreadIdx = alpaka::getIdx<alpaka::Grid, alpaka::Threads>(acc)[0];
-        int const gridThreadExtent = alpaka::getWorkDiv<alpaka::Grid, alpaka::Threads>(acc)[0];
+        using Dim = alpaka::Dim<TAcc>;
+        using Idx = alpaka::Idx<TAcc>;
+        using Vec = alpaka::Vec<Dim, Idx>;
```

Comment:
The initial definitions in L10-12 are temporary. They should be moved to a more appropriate file eventually. The definitions here are good practice I think, since these parameters are taken directly from the TAcc input rather than some more global scope.

Reply:
Thanks, I see. Then maybe delete L10-12, since they are bad practice?

Reply:
Those need to be there for now, since they are used to create Queues. Eventually they will be moved to a more appropriate file, though.